src/components/QuizSettings.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { withStyles } from "@material-ui/core/styles";
+import ToggleButton from "@material-ui/lab/ToggleButton";
+import ToggleButtonGroup from "@material-ui/lab/ToggleButtonGroup";
+
+const StyledToggleButton = withStyles({
+  root: {
+    background: "none",
+    borderRadius: 4,
+    color: "black",
+    height: 42,
+    width: 150,
+    padding: "0 30px",
+    borderColor: "#d4d4d4",
+    "&:hover": {
+      borderColor: "#21B6A8",
+      background: "none",
+    },
+  },
+  selected: {
+    "&&": {
+      borderColor: "#21B6A8",
+      borderWidth: "medium",
+      background: "none",
+      color: "black",
+      "&:hover": {
+        background: "none",
+        borderColor: "#21B6A8",
+      },
+    },
+  },
+})(ToggleButton);
+
+export const QuizSettings = (props) => {
```

Comment:
A small thing you should know about props in React components (you can continue using your current approach if you like, but thought you should know this): 

It's possible to use destructuring to list all the properties as arguments here instead of just having one "props" object :)

```js
// Before
export const QuizSettings = (props) => {
  props.setInterruption(number);
  console.log(props.interruption);
}

// After
export const QuizSettings = ({ interruption, setInterruption }) => {
  setInterruption(number);
  console.log(interruption);
}
```

The best thing about this approach is that you make it very clear what properties the component is expecting, whereas with the `props` approach you have to lookup the parent components to figure out what properties are available.